azure-pipelines.yml

```diff
@@ -541,7 +564,7 @@ stages:
           options: -Dtest=$(testClasses) -B --settings $(mavenSettings.secureFilePath)
           publishJUnitResults: true
           testResultsFiles: '**/surefire-reports/TEST-*.xml'
-          testRunTitle: 'ITs on Windows'
+          testRunTitle: 'ITs on Windows $(Agent.JobName)'
```

Comment:
This is to distinguish this two runs:
![image](https://user-images.githubusercontent.com/103252490/182840520-f618249a-3c6a-48e3-848d-5a9fb7a4131b.png)

Reply:
Nice

Reply:
I would remove the `ITs on Windows`. The JobName is way better and clear by itself.

Reply:
Please do the same trick on `UTs on Windows` too, because JobName has better and more clear value